src/components/PersonLink.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  names: string[];
+  findParent: (el: string) => Person | undefined;
+}
+
+export const PersonLink: React.FC<Props> = ({ person, names, findParent }) => {
+  const { slug } = useParams();
+
+  return (
+    <>
+      <tr
+        data-cy="person"
+        className={cn({ 'has-background-warning': person.slug === slug })}
+      >
+        <td>
+          <Link
+            className={cn({ 'has-text-danger': person.sex === 'f' })}
+            to={`/people/${person.slug}`}
+          >
+            {person.name}
+          </Link>
+        </td>
+
+        <td>{person.sex}</td>
+        <td>{person.born}</td>
+        <td>{person.died}</td>
+        <td>
+          {person.motherName && names.includes(person.motherName) ? (
+            <Link
+              className="has-text-danger"
+              to={`/people/${findParent(person.motherName)?.slug}`}
```

Comment:
The `findParent` function is used to retrieve the parent's slug, but it might return `undefined` if the parent is not found. Consider adding a check to ensure that the `slug` is defined before constructing the URL to prevent potential issues with invalid URLs.